tools/Customization/DevHome.Customization/Strings/en-us/Resources.resw

```diff
@@ -161,9 +161,9 @@
     <value>Enable end task in taskbar by right click</value>
     <comment>The description for the end task on task bar settings card</comment>
   </data>
-  <data name="ExampleDevDriveLocation" xml:space="preserve">
-    <value>Example: E:\packages\pip</value>
-    <comment>Example dev drive location</comment>
+  <data name="ExampleText" xml:space="preserve">
+    <value>Example: </value>
+    <comment>Example string</comment>
```

Comment:
NIT: Add more to the comments to differentiate this from an example.  I thought this was a sample copy and pasted from online.

Reply:
Updated.